adversarial_apps/src/app/education/cfr-title-15/page.tsx

```diff
@@ -3,8 +3,8 @@ export default function page(){
     <>
         <main>
             <h1 className="text-6xl font-bold pl-5">CFR Title 15</h1>
-            <h4 className="text-xl font-bold pl-5"><br />What is Title 15</h4>
-            <p className = "pl-5">A CFR stands for Code of Federal Regulations and contains 
+            <h4 className="text-xl font-bold pl-5"><br />What is Title 15 about?</h4>
```

Comment:
I would look into changing this to "h2". Ideally we don't want to skip header numbers. You can check the Home Page I think to see what this looks like regarding headers.